products/logs/frontend/components/LogsTable/LogsTableRowActions.tsx

```diff
@@ -1,16 +1,34 @@
+import { useValues } from 'kea'
+
 import { IconCopy } from '@posthog/icons'
 
 import { LemonButton } from 'lib/lemon-ui/LemonButton'
 import { More } from 'lib/lemon-ui/LemonButton/More'
+import { IconLink } from 'lib/lemon-ui/icons'
 import { copyToClipboard } from 'lib/utils/copyToClipboard'
 
 import { LogMessage } from '~/queries/schema/schema-general'
 
+import { logsLogic } from 'products/logs/frontend/logsLogic'
+
 interface LogsTableRowActionsProps {
     log: LogMessage
 }
 
 export function LogsTableRowActions({ log }: LogsTableRowActionsProps): JSX.Element {
+    const { visibleLogsTimeRange } = useValues(logsLogic)
+
+    const handleCopyLink = (): void => {
+        const url = new URL(window.location.href)
+        url.searchParams.set('highlightedLogId', log.uuid)
```

Comment:
thinking about it i think we need to populate all filters here, it's very unlikely with our prod volume that the highlighted log would be visible after clearing the filters (we'll be showing 100 logs out of like 4+Million per minute)